service/src/main/java/io/camunda/service/ManagementServices.java

```diff
@@ -9,15 +9,16 @@
 
 import io.camunda.service.license.CamundaLicense;
 
-public final class ManagementService {
+public final class ManagementServices {
 
-  private final CamundaLicense camundaLicense;
+  private CamundaLicense camundaLicense;
 
-  public ManagementService() {
-    camundaLicense = new CamundaLicense();
+  public ManagementServices withLicense(final CamundaLicense license) {
```

Comment:
better use the constructor to make `ManagementServices` immutable and avoid nullity checks on `camundaLicense` (I don't know in which functional case it can be null?)

if you want to use `withLicense` pattern, you can do 
```java
public ManagementServices withLicense(final CamundaLicense license) {
return new ManagementServices(license);
}

public ManagementService() {
}

public ManagementService(CamundaLicense license) {
    camundaLicense= license;
}
```